File: submissions/sm_022_prakash/week_03/day_3/submissions_2/average_divisible.js
sum = 0;
number = 20;
count = 0;
//function to print average of all the numbers divisible by given numbers 
function average(number) {
  for (i = 1; i <= number; i++) {
    if (i % 2 == 0) {
      // console.log(i)
      sum = sum + i;
      count = count + 1;
      // console.log(sum)
      average = sum / count;
      // console.log(average)
    }
  }
  return average;
}
average(20);
console.log(average);

File: submissions/sm_107_hasmuddin/week_13/day_4/session_1/average_divisible.py
''' Print the average of all the
numbers from 0 to the given limit that are divisible by the given number'''
limit = int(input("limit: "))
num = int(input("Number: "))
sum = 0
count = 0
for i in range(limit):
    if i%num ==0:
        sum += i
        count += 1
ave = sum/count
print(ave)